js/bitvavo.js

```diff
@@ -14,7 +14,7 @@ module.exports = class bitvavo extends Exchange {
             'id': 'bitvavo',
             'name': 'Bitvavo',
             'countries': [ 'NL' ], // Netherlands
-            'rateLimit': 60.1, // 1000 requests per second
+            'rateLimit': 60, // 1000 requests per minute
```

Comment:
@frosty00 I think 60.1 should have been there for same rate limits. @kroitor right?